.github/workflows/parallel_tests.yml

```diff
@@ -1,4 +1,4 @@
-name: parallel_tests
+name: continuous-integration/github/pr
```

Comment:
(Left my comment in the wrong place!) 
This is a new workflow. What is the added value?